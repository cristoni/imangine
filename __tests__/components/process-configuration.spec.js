import React from 'react';
import ProcessConfiguration from '../../components/process-configuration';

test('ProcessConfiguration renders the process configuration panel for uploaded images', () => {
  let wrapper = shallow(<ProcessConfiguration />);
  expect(wrapper).toMatchSnapshot();

  const mockFiles = [
    { name: 'file1' },
    { name: 'file2' },
    { name: 'file3' }
  ];

  wrapper = shallow(<ProcessConfiguration files={mockFiles} />);
  expect(wrapper).toMatchSnapshot();
});