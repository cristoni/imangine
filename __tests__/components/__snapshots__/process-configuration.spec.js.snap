// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProcessConfiguration renders the process configuration panel for uploaded images 1`] = `
<div
  className="process-configuration"
>
  <h4
    className="process-configuration__title"
  >
    Process 
    <span
      className="hl"
    >
      C
    </span>
    onfiguration
  </h4>
  <hr />
  <Formik
    enableReinitialize={false}
    initialValues={
      Object {
        "crop": false,
        "height": 1080,
        "resolution": false,
        "width": 1920,
      }
    }
    isInitialValid={false}
    onSubmit={[Function]}
    validate={[Function]}
    validateOnBlur={true}
    validateOnChange={true}
  >
    <Component />
  </Formik>
</div>
`;

exports[`ProcessConfiguration renders the process configuration panel for uploaded images 2`] = `
<div
  className="process-configuration"
>
  <h4
    className="process-configuration__title"
  >
    Process 
    <span
      className="hl"
    >
      C
    </span>
    onfiguration
  </h4>
  <hr />
  <Formik
    enableReinitialize={false}
    initialValues={
      Object {
        "crop": false,
        "height": 1080,
        "resolution": false,
        "width": 1920,
      }
    }
    isInitialValid={false}
    onSubmit={[Function]}
    validate={[Function]}
    validateOnBlur={true}
    validateOnChange={true}
  >
    <Component />
  </Formik>
</div>
`;
